src/blurb/blurb.py

```diff
@@ -869,10 +876,112 @@ def find_editor():
     error('Could not find an editor! Set the EDITOR environment variable.')
 
 
+def _extract_issue_number(issue):
+    if issue is None:
+        return None
+
+    issue = raw_issue = str(issue).strip()
+    if issue.startswith('gh-'):
+        issue = issue[3:]
+    if issue.isdigit():
+        return issue
+
+    match = re.match(r'^(?:https?://)?github\.com/python/cpython/issues/(\d+)$', issue)
```

Comment:
Do you think someone might copy and paste an HTTP URL nowadays?

```suggestion
    match = re.match(r'^(?:https://)?github\.com/python/cpython/issues/(\d+)$', issue)
```

Reply:
You're right. I don't think there are any browsers that would keep HTTP over HTTPS. I think github also redirects automatically to HTTPS.